integration/tests/cook/util.py

```diff
@@ -1194,22 +1194,53 @@ def slave_pool(cook_url, mesos_url, hostname):
     return pool
 
 
-def max_slave_cpus(mesos_url):
+def max_mesos_slave_cpus(mesos_url):
     """Returns the max cpus of all current Mesos agents"""
     slaves = get_mesos_state(mesos_url)['slaves']
     max_slave_cpus = max([s['resources']['cpus'] for s in slaves])
     return max_slave_cpus
 
+def get_kubernetes_nodes():
+    cook_url = retrieve_cook_url()
+    compute_clusters = settings(cook_url)['compute-clusters']
+    kubernetes_compute_cluster = [cc for cc in compute_clusters
```

Comment:
Refactor this section out and re-use it for using_kubernetes()?

Reply:
Done